src/mantine-core/src/components/Skeleton/Skeleton.styles.ts

```diff
@@ -0,0 +1,46 @@
+import { createMemoStyles, DefaultProps } from '../../theme';
+
+export interface SkeletonProps extends DefaultProps {
+  children?: React.ReactNode;
```

Comment:
- Rename this interface to SkeletonStyles
- Remove `extends DefaultProps`
- Remove all unused props, currently you use `height, type, animated, startColor, endColor`